frontend/src/components/Util/GroupMemberListPicker/SearchGroupMember.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { ReactElement } from 'react';
+import Fuse from 'fuse.js';
+import { GroupMember } from 'common/types/store-types';
+import GroupMemberSearchBox from '../GroupMemberSearchBox/GroupMemberSearchBox';
+import styles from './SearchGroupMember.module.scss';
+
+type SimpleMember = {
+  readonly key: number;
+  readonly value: string; // name of student
+  readonly id: string; // netId of student
+};
+
+type Props = { readonly members: GroupMember[] };
+
+/**
+ * Returns computed group member options.
+ *
+ * @param {GroupMember[]} group members.
+ * @return {SimpleMember[]} group member options.
+ */
+function getMemberOptions(members: GroupMember[]): SimpleMember[] {
+  const memberOptions: SimpleMember[] = [];
+  let i = 0;
+  members.forEach((member: GroupMember) => {
+    const { netId, name } = member;
+    const id = netId;
+    memberOptions.push({
+      key: i,
+      value: name,
+      id,
+    });
+    i += 1;
```

Comment:
instead of keeping track of a mutable variable and incrementing it, we can use the second argument of `.forEach()` to get indices of `members`.